ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -362,6 +362,18 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec
+- as: azure-ipi-private-workers-marketimage-p2-f7
```

Comment:
Suggest to lower the test frequency a bit to f14.

Reply:
What's your suggested cron setting? @jianlinliu

Reply:
cron setting is supposed to be re-generated by running https://raw.githubusercontent.com/openshift/release/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh after you correct the test name to `azure-ipi-private-workers-marketimage-p2-f7`.

Reply:
Ok, I will update it against my thought.

Reply:
@jianlinliu done, please take a look again